qiita_pet/uimodules/prep_template_tab.py

```diff
@@ -218,7 +219,19 @@ def render(self, study, prep_template, full_access, ena_terms,
                         no_preprocess_msg = (
                             "Preprocessing disabled due to missing columns in "
                             "the prep template: %s" % ', '.join(missing_cols))
-        preprocessing_status = 'TODO: plugin'
+
+            # Check the processing status
+            jobs = raw_data.jobs(status='running')
```

Comment:
I almost feel like this block of logic should be in the artifact class, and not in here, what do you think?